### PR TITLE
Ria 3488

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-3488

### Change description ###

PAY OFFLINE-Notification when a EA, HU appeal has been submitted with payment pending

**Does this PR introduce a breaking change?** (check one with "x")

```
[ X] Yes
[ ] No
```

### CODE REVIEW

**Review by Lee365 — CHANGES_REQUESTED** [review]
New State is 'pendingPayment'

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/ccd/State.java [hunk 1]
```diff
@@ -8,6 +8,7 @@
     APPEAL_STARTED("appealStarted"),
     APPEAL_SUBMITTED("appealSubmitted"),
     APPEAL_SUBMITTED_OUT_OF_TIME("appealSubmittedOutOfTime"),
+    PAYMENT_PENDING("paymentPending"),
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/ccd/State.java:11]: New state is 'pendingPayment'

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java [hunk 2]
```diff
@@ -369,7 +395,8 @@
                 return callbackStage == PreSubmitCallbackStage.ABOUT_TO_SUBMIT
                     && callback.getEvent() == Event.SUBMIT_APPEAL
```
Comment (Lee365) [inline src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java:396]: Does this need to include check for PAY_AND_SUBMIT_APPEAL

**Review by Lee365 — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/ccd/State.java (hunk 1 above)
Comment (mkayad) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/ccd/State.java:11]: ok i will change

Comment (mkayad) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/entities/ccd/State.java]: State name is changed

File: src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java (hunk 2 above)
Comment (mkayad) [reply src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/infrastructure/config/NotificationHandlerConfiguration.java:396]: It is not required.

**Review by Lee365 — DISMISSED** [review]
LGTM

**Review by Lee365 — APPROVED** [review]
LGTM

**Review by Srihari-Nakkala — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**